api/models/settings.py

```diff
@@ -0,0 +1,228 @@
+from api.db import db
+
+
+class ApplicationEnvironament:
+
+    def __init__(self):
+        pass
+    DEVELOPMENT = 'development'
+    STAGING = 'staging'
+    PRODUCTION = 'production'
+    TESTING = 'testing'
+
+
+class Settings(db.Model):
+    __tablename__ = 'settings'
+
+    app_environment = db.Column(db.String, default=applicationEnvironament.PRODUCTION)
+    # Name of the application. (Eg. Badgeyay)
+    app_name = db.Column(db.String)
+    # Description of the App
+    app_description = db.Column(db.String(140))
```

Comment:
Where do you think it will be beneficial to store, as if you also use the description in accessing keys from gateway them also you have to either write or manually copy paste the thing

Reply:
@yashLadhaI am not clear what you were asking? I added this which we directly populate the description box above `create badges`